packages/react-card/src/components/CardHeader/CardHeader.types.ts

```diff
@@ -3,10 +3,10 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 export type CardHeaderSlots = {
   root: Slot<'div'>;
   image: Slot<'div'>;
-  content?: Slot<'div'>;
+  content?: Slot<'div'> | null;
   header: Slot<'span'>;
   description: Slot<'span'>;
```

Comment:
Note: the `image`, `header`, and `description` slots were previously nullable, but with this change they are non-nullable (as a result of `Slot` no longer including `| null`). It appears from the code that they were never intended to be nullable.